### DISCUSSION TITLE
Got Another NFT!!!!.......Third one......

### DISCUSSION THREAD

**Discussion by Raghav-Programmer** [Cyfrin/foundry-full-course-cu#349] [General]
I completed lesson-3 and solved the challenge...Can't wait to dive deeper into solidity and the web3 world....Thanks @PatrickAlphaC for creating this awesome course.....

![nft](https://github.com/Cyfrin/foundry-full-course-f23/assets/113235744/b567a602-742b-44d9-a86b-bccc8a005568)

**Comment by IAm0xAxe** [comment]
Congratulations.

**Reply by Raghav-Programmer** [reply]
Thanks! @IAm0xAxe
